packages/opencensus-web-all/src/types.ts

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright 2019, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {PerformanceLongTaskTiming, WindowWithLongTasks} from '@opencensus/web-instrumentation-perf';
+
+/** OpenCensus Web configuration that can be set on `window`. */
+export declare interface OpenCensusWebConfig {
+  /**
+   * Whether the initial load root span should be sampled for trace. This is
+   * only a hint and should not be used to enforce sampling since clients could
+   * simply ignore this field.
+   */
+  sampled?: boolean;
+  /** Trace ID for the initial load spans. */
+  traceId?: string;
+  /** Span ID for the initial load fetch client span. */
+  spanId?: string;
+  /**
+   * Start time of server fetch request for initial navigation HTML in server
+   * time epoch milliseconds. This is used to correct for clock skew between
+   * client and server before exporting spans.
+   */
+  reqStart?: number;
```

Comment:
Can this be a ```reqStartTime```?

Reply:
Sure, done. Per design convos, this might get removed, but let's do that once we clarify the design and not block this overall PR on it.